src/safe_apps/migrations/0009_safe_apps_features.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.1.5 on 2023-01-12 16:35
+from django.apps.registry import Apps
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations import RunPython
+
+
+def add_default_features(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    Feature = apps.get_model("safe_apps", "Feature")
+    Feature.objects.create(key="BATCHED_TRANSACTIONS")
```

Comment:
I'm not sure if the migration itself warrants a single default Safe App `Feature`. Given that this project is also open-source I'd probably leave our own use-case out from the migration and add it via the admin. Wdyt?